interval/__init__.py

```
@@ -4,33 +4,58 @@
+"""
+
 import numpy as np
 
 try:
     from ._version import __version__
-except ImportError as err:
+except ImportError:
     __version__ = 'Version unspecified'
 
 
 class Interval(object):
```

Ruff (0.14.6)

Class `Interval` inherits from `object`

Remove `object` inheritance

(UP004)